pkg/controllers/disruption/controller.go

```diff
@@ -114,6 +114,9 @@ func (c *Controller) Reconcile(ctx context.Context, _ reconcile.Request) (reconc
 	// with making any scheduling decision off of our state nodes. Otherwise, we have the potential to make
 	// a scheduling decision based on a smaller subset of nodes in our cluster state than actually exist.
 	if !c.cluster.Synced(ctx) {
+		if time.Since(c.cluster.LastSyncTime()) > state.AbnormalClusterSyncDuration { 
+			logging.FromContext(ctx).Infof("cluster state has not been in sync since %v, waiting for sync", c.cluster.LastSyncTime()) 
```

Comment:
Alternatively we could log how many seconds it has been out of sync. Or we can do some modulo math to log this only every 10 seconds. The behavior here is we will log out of sync since <time> over and over until its synced